src/status_im/common/signals/events.cljs

```diff
@@ -70,10 +69,10 @@
       "messages.new"               (messages.transport/sanitize-messages-and-process-response cofx
                                                                                               event-js
                                                                                               true)
-      "wallet"                     (ethereum.subscriptions/new-wallet-event cofx
-                                                                            (js->clj event-js
-                                                                                     :keywordize-keys
-                                                                                     true))
+      "wallet"                     (rf/dispatch [:wallet/signal-fired
+                                                 (js->clj event-js
+                                                          :keywordize-keys
+                                                          true)])
```

Comment:
The `js->clj` method from `utils.transforms` is easy to use and has the extra `:keywordize-keys` config handled.